README.md

```diff
@@ -4,46 +4,57 @@ This repository contains additional **C++** code samples for **Zivid**.
 
 [![Build Status][ci-badge]][ci-url]
 
-The basic samples are available at [https://www.zivid.com/downloads](https://www.zivid.com/downloads).
-The Windows Zivid installer adds these samples in C:\Users\Public\Documents\Zivid\samples and they should build out of the box using Visual Studio 2015 or 2017.
 Check out our [tutorial on configuring and building these samples on Ubuntu](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/59441336/Configure+C+Samples+with+CMake+and+then+build+them+using+make+in+Ubuntu).
```

Comment:
For better readability of readme in plain text mode you can add urls to the end of the file like seen above. If you agree that is an improvement..